demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
+  width:auto;
+  height:82px;
+  margin-left:35%;
+  margin-top: 7px;
+}
+
+@media (min-width: 576px){
```

Comment:
2. Then, remove this non-mobile `@media` and put `.videw-preview` outside. As a result, we can rely on one `@media` query.

Reply:
yeah sounds good